src/signals/incident/components/form/Header/index.js

```diff
@@ -60,7 +58,9 @@ const Header = ({ className, meta, options, touched, hasError, getError, childre
 
       {touched && containsErrors && (
         <Fragment>
-          {hasError('required') && <ErrorItem>{getError('required') || 'Dit is een verplicht veld'}</ErrorItem>}
+          {hasError('required') && (
+            <ErrorItem>{getError('required') === true ? 'Dit is een verplicht veld' : getError('required')}</ErrorItem>
```

Comment:
This is confusing, I have to say. It can be either a boolean or a string?

Reply:
I agree it's confusing - but in case of an error, this is what getError of react-reactive-form returns

Reply:
I see, it's because of how react-reactive-form works.